analyzers/its/expected/NetCore31WithConfigurableRules/NetCore31WithConfigurableRules--netcoreapp3.1.Views-S1128.json

```diff
@@ -82,32 +56,6 @@
 "id":  "S1128",
 "message":  "Remove this unnecessary 'using'.",
 "location":  {
-"uri":  "https://github.com/SonarSource/sonar-dotnet/blob/master/analyzers/its/sources/NetCore31WithConfigurableRules/obj/Debug/netcoreapp3.1/Razor/Views/_ViewStart.cshtml.g.cs#L17",
```

Comment:
Have you checked where some of these issues disappeared? My guess is that they did not have mapping in the generated file so we skipped them?

Reply:
As discussed, those issues will disappear once #7959 is implemented.